PCGen-base/gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
need to revert this

Reply:
why? It is needed for windows and does not affect functioning of the application.

Reply:
It's limiting the max memory to 64M - PCGen won't run

Reply:
affects gradle, not pcgen